backend/layers/thirdparty/cloudfront_provider_interface.py

```diff
@@ -0,0 +1,3 @@
+class CloudfrontProviderInterface:
```

Comment:
nit: technically Cloudfront is the implemented provider, so if we want to name a generic provider interface it should be CDNProvider or similar